packages/webview_flutter/webview_flutter_wkwebview/lib/src/foundation/foundation.dart

```diff
@@ -226,7 +243,15 @@ class NSError {
   final String domain;
 
   /// A string containing the localized description of the error.
-  final String localizedDescription;
+  String? get localizedDescription =>
+      userInfo[NSErrorUserInfoKey.NSLocalizedDescription] as String?;
```

Comment:
This actually nullable in the Objc `NSError` and is always equal to this value in the `userInfo` map.